installer/win/util/brave_vpn_helper_utils.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_INSTALLER_WIN_UTIL_BRAVE_VPN_HELPER_UTILS_H_
+#define BRAVE_INSTALLER_WIN_UTIL_BRAVE_VPN_HELPER_UTILS_H_
+
+#include <windows.h>
+#include <string>
+
+#include "base/files/file_path.h"
+#include "base/version.h"
+
+namespace brave_vpn {
+
+bool ConfigureServiceAutoRestart(const std::wstring& service_name,
+                                 const std::wstring& brave_vpn_entry);
+base::FilePath GetBraveVpnHelperServicePath();
+base::FilePath GetVpnHelperServiceProfileDir();
+bool InstallBraveVPNHelperService();
+bool IsBraveVPNHelperServiceInstalled();
+bool IsNetworkFiltersInstalled();
+HRESULT InstallBraveVPNHelperServiceImpersonated();
+HRESULT InstallBraveWireGuardServiceImpersonated();
+std::wstring GetBraveVPNConnectionName();
+std::wstring GetBraveVpnHelperServiceDisplayName();
+std::wstring GetBraveVpnHelperServiceName();
+
+bool InstallVPNSystemServices();
```

Comment:
why is this in installer? It doesn't appear to actually be used in the installer

Reply:
Broke out to where it was getting called from - see https://github.com/brave/brave-core/pull/20754/commits/382b47b4e8cac60ab4ff17617873afeb4bc82905

Reply:
the method declaration is still here, but there is no definition or anything calling it